db/schema.rb

```diff
@@ -54,8 +54,8 @@
     t.string   "experience"
     t.string   "interest"
     t.string   "school_id"
-    t.datetime "created_at",                                   :null => false
-    t.datetime "updated_at",                                   :null => false
+    t.datetime "created_at"
+    t.datetime "updated_at"
```

Comment:
So, the default for datetime fields should be 0 (not null), but I feel like Rails might have implemented this itself being its defined on MySQL's reference docs. Gonna check later tonight

http://dev.mysql.com/doc/refman/5.7/en/data-type-defaults.html